webapp/src/Controller/Jury/LanguageController.php

```diff
@@ -140,10 +136,10 @@ public function indexAction(): Response
 
     // Note that the add action appears before the view action to make sure
     // /add is not seen as a language.
-    /**
-     * @Route("/add", name="jury_language_add")
-     * @IsGranted("ROLE_ADMIN")
-     */
+    #[IsGranted('ROLE_ADMIN')]
+    // Note that the add action appears before the view action to make sure
```

Comment:
This comment is now duplicated

Reply:
Thanks for catching. Rector has a bug where this happens and I thought I got all of them